cli/util.go

```diff
@@ -388,7 +388,7 @@ func natsOpts() []nats.Option {
 	kingpin.FatalIfError(err, "configuration error")
 
 	return append(copts, []nats.Option{
-		nats.Name("NATS CLI Version " + Version),
+		nats.Name(opts.ConnectionName),
```

Comment:
Above this check if `opts.ConnectionName` is empty and set it to default, that way even when embedded the cli behavior wont change

Reply:
Done.